### ISSUE TITLE
Ending support for Node.js 14 in AWS Lambda

### ISSUE THREAD

**Issue by jacekhewko** [awslabs/landing-zone-accelerator-on-aws#248]
Hi there,

Just informational - we are receiving Node.js 14 end-of-life for AWS Lambda e-mails, and it turns out it concerns LZA solution:

<img width="1382" alt="image" src="https://github.com/awslabs/landing-zone-accelerator-on-aws/assets/50452852/51a49ade-889f-474f-8813-9b27a36af33d">

Are you able to tell already when is it planned to be updated?

```

Hello,

We are contacting you as we have identified that your AWS Account currently has one or more Lambda functions using the Node.js 14 runtime.

We are ending support for Node.js 14 in AWS Lambda. This follows Node.js 14 End-Of-Life (EOL) reached on April 30, 2023 [1].

As described in the Lambda runtime support policy [2], end of support for language runtimes in Lambda happens in two stages. Starting November 27, 2023, Lambda will no longer apply security patches and other updates to the Node.js 14 runtime used by Lambda functions, and functions using Node.js 14 will no longer be eligible for technical support. In addition, you will no longer be able to create new Lambda functions using the Node.js 14 runtime. Starting January 25, 2024, you will no longer be able to update existing functions using the Node.js 14 runtime.

We recommend that you upgrade your existing Node.js 14 functions to Node.js 18 before November 27, 2023.

End of support does not impact function execution. Your functions will continue to run. However, they will be running on an unsupported runtime which is no longer maintained or patched by the AWS Lambda team.

This notification is generated for functions using the Node.js 14 runtime for the $LATEST function version. For a list of your affected Lambda functions, please see the 'Affected resources" tab of your AWS Health Dashboard

The following command shows how to use the AWS CLI [3] to list all functions in a specific region using Node.js 14, including published function versions. To find all such functions in your account, repeat this command for each region:

aws lambda list-functions --function-version ALL --region us-east-1 --output text --query "Functions[?Runtime=='nodejs14.x'].FunctionArn"

If you have any concerns or require further assistance, please contact AWS Support [4].


```

Thanks,
Jacek

**Comment by bhkhatri221** [comment]
Hi @jacekhewkom, thank you for creating this issue! We have implemented the fix for this issue, and it  will be included in our next release. To reflect the nature of this improvement, I've changed the tag from "bug" to "enhancement".

Once again, thank you for your valuable input.

**Comment by johnraws** [comment]
Hello @jacekhewko, this fix is being tracked as part of #240 in the upcoming release. Please follow that issue for any updates regarding the implementation of the fix. 

This issue will be closed in favor of #240

**Comment by bugzzbunny007** [comment]
it is taking time to upgrade all the lambda's present on the aws is there any issue if it takes some more time after 25 Jan 2024 Will i be able to upgrade it easily or need to create new lambda's ?
